classes/controller/AdminController.php

```diff
@@ -1521,6 +1521,13 @@ protected function processUpdateOptions()
                         $this->errors[] = $this->trans('field %s is required.', [$values['title']], 'Admin.Notifications.Error');
                     }
                 }
+                
+                // Check if field value on type select is valid
+                if (isset($values['type']) && $values['type'] == 'select' && isset($values['identifier']) && !empty($values['list'])) {
+                    if (false === in_array(Tools::getValue($field), array_column($values['list'], $values['identifier']))) {
+                        $this->errors[] = $this->trans('The %s field is invalid.', [$values['title']], 'Admin.Notifications.Error');
```

Comment:
Code seems OK, but should we be a little more accurate? Something like `The %s option you have selected is invalid / cannot be selected` ?

we can ask @Julievrz

Reply:
I suggest the following error message:

'Only the items in the drop-down list can be selected. Please remove "%s%" to continue.' in Admin.Notifications.Error

Reply:
`%s` is the field not the option. I think `Please remove "%s%" to continue.` is not right.
Juste simply: `The option selected in the %s field is invalid.` ?

Reply:
I think it would be clearer if the variable was the invalid selected option, such as:

'Only the items in the drop-down list can be selected. Please remove "%InvalidSelectedOption%" to continue.'

Reply:
The option is invalid, there isn't named. Only the hidden value is accessible, so the error will be for exemple:
- `Only the items in the drop-down list can be selected. Please remove "99999" to continue.`
- `Only the items in the drop-down list can be selected. Please remove "<script>alert('test')</script>" to continue.`

Reply:
Ok, I see. 
We can go with 'The option selected in the %s field is invalid.' in Admin.Notifications.Error then.